ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/PaymentIntentionMapper.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPaymentOption;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPaymentSchedule;
+import uk.gov.hmcts.cmc.ccd.domain.response.CCDPaymentIntention;
+import uk.gov.hmcts.cmc.domain.models.PaymentOption;
+import uk.gov.hmcts.cmc.domain.models.RepaymentPlan;
+import uk.gov.hmcts.cmc.domain.models.ccj.PaymentSchedule;
+import uk.gov.hmcts.cmc.domain.models.response.PaymentIntention;
+
+
```

Comment:
please remove extra line

Reply:
removed. I suggest in the future you add any of these rules to checkstyle it will save us both a lot of time and effort. we just can't keep making formatting rules on the fly.

Reply:
FYI- Agreed ways of working in team has all this here:
https://tools.hmcts.net/confluence/display/ROC/Ways+of+Working+-+Tech+Team